apps/users/src/app/controllers/users.controller.v2.ts

```diff
@@ -87,12 +87,12 @@ export class UsersControllerV2 {
     @Query() userQuery?: ExpansionQuery
   ) {
     const user = isMe
-      ? authorizer.user
+      ? await authorizer.user
       : await this.userService.findUser(userId);
 
     return this.userServiceV2.getById(
```

Comment:
จากที่ดู function นี้ param ตัวแรกคือ คนที่ request เข้ามา ส่วนตัวที่สอง คือ คนที่ถูก request ข้อมูล​ ถูกไหมอะ?
1. ถ้าส่ง user, user._id ไป มันก็เป็นคนเดียวกันตลอด
2. ถ้า requester A -> ส่ง userId: B -> parameter user ในบรรทัด 89 ก็จะเป็น B แสดงว่า relationship ที่ออกมา ก็จะกลายเป็นของ B กับ B แทน

ดังนั้น param1=authorized user, param2=user to lookup